src/plugins/dashboard/public/dashboard_container/embeddable/api/duplicate_dashboard_panel.ts

```diff
@@ -56,8 +57,8 @@ const duplicateReactEmbeddableInput = async (
   panelToClone: DashboardPanelState,
   idToDuplicate: string
 ) => {
-  const child = dashboard.reactEmbeddableChildren.value[idToDuplicate];
-  if (!child) throw new PanelNotFoundError();
+  const child = dashboard.children$.value[idToDuplicate];
+  if (!child || !apiHasSerializableState(child)) throw new PanelNotFoundError();
```

Comment:
should failed apiHasSerializableState throw a different error then PanelNotFoundError? Maybe IncompatablePanelFoundError?

Reply:
Good idea. I'm sure we'll need plenty of guards that enforce the validity of a panel type. So I've created a new `PanelIncompatibleError` error type in the embeddable plugin in cde847b61cf6